app/ts/background/simulationModeHanders.ts

```diff
@@ -70,8 +70,8 @@ export async function sendTransaction(
 			from,
 			chainId: ethereumClientService.getChainId(),
 			nonce: await transactionCount,
-			maxFeePerGas: transactionDetails.maxFeePerGas != undefined ? transactionDetails.maxFeePerGas : parentBlock.baseFeePerGas * 2n,
-			maxPriorityFeePerGas: transactionDetails.maxPriorityFeePerGas != undefined  ? transactionDetails.maxPriorityFeePerGas : 10n**8n, // 0.1 nanoEth/gas
+			maxFeePerGas: transactionDetails.maxFeePerGas !== undefined && transactionDetails.maxFeePerGas !== null ? transactionDetails.maxFeePerGas : parentBlock.baseFeePerGas * 2n,
+			maxPriorityFeePerGas: transactionDetails.maxPriorityFeePerGas !== undefined && transactionDetails.maxPriorityFeePerGas !== null ? transactionDetails.maxPriorityFeePerGas : 10n**8n, // 0.1 nanoEth/gas
```

Comment:
I wonder if it is worth adding a comment indicating that EtherScan sets these to null sometimes which is why we have them, that way if they eventually fix the bug we can go back and delete this cruft.

Reply:
I added comment to the type